vscode-trace-webviews/src/trace-explorer/opened-traces/vscode-trace-explorer-opened-traces-widget.tsx

```diff
@@ -86,6 +86,11 @@ class TraceExplorerOpenedTraces extends React.Component<{}, OpenedTracesAppState
                       this.setState({experimentsOpened: true});
                   }
               }
+              break;
+          case VSCODE_MESSAGES.TRACE_SERVER_STARTED: {
```

Comment:
Do we need to use the curly bracket to group the code block here? It is not done in other block statements. Could you format this block to match the other blocks of the switch statements?